talk-4-advanced-js-2/README.md

```diff
@@ -11,8 +11,13 @@
 ### Callbacks
 - 
 
+### TCO
+- As it stands, JavaScript is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in `Maximum call stack size exceeded`
```

Comment:
> is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls.

I'm not sure if this is supposed to be notes or the basis for slides, but that's a little repetitive.